src/components/ViewCourse/sections/ViewCourseData.js

```diff
@@ -37,15 +37,18 @@ const ViewCourseData = ({ record }) => {
           <Col xs={3}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.name" />} value={courseNameAndDescription} />
           </Col>
-          <Col xs={3}>
+          <Col xs={2}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.department" />} value={departmentNameAndDescription} />
           </Col>
-          <Col xs={3}>
+          <Col xs={2}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.number" />} value={record.courseNumber} />
           </Col>
-          <Col xs={3}>
+          <Col xs={2}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.section" />} value={record.sectionName} />
           </Col>
+          <Col xs={3}>
```

Comment:
Seems weird to give this integer a wider space than any of courseNameAndDescription, courseNumber or sectionName.